cpu arch support changes

jyecusch opened the pull request.

jyecusch committed:
fix: remove i386 binary releases

jyecusch committed:
fix: error when running `up` on non-x86_64 CPUs

Using other CPU architectures (e.g. ARM) causes compatibility issues with the built containers currently. We'll fix this limitation, but in the meantime we want to present a more useful error message.

asalkeld commented on pkg/cmd/stack/root.go:
```diff
@@ -18,14 +18,14 @@ package project
+	"log"
+	"os"
+	"path/filepath"
+	"runtime"
```
you probably need to revert these import changes

jyecusch committed:
ci: only upload codecov report once

We were uploading the Codecov report one for each tested OS. This issue is currently preventing the MacOS upload https://github.com/codecov/codecov-action/issues/745
We don't need both, so I'm filtering out the mac version of this step
